common/decisions/90_minor_decisions.txt

```diff
@@ -950,7 +950,7 @@ stress_loss_flagellant_decision = { #by Linnéa Thimrén
 		custom_tooltip = stress_loss_flaggelant_decision_effect_tooltip
```

Comment:
I think you should say it's stackable. Also, I would consider increasing years to 20 since it's hard to get a good amount of successful schemes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/119344415-b14be200-bca8-11eb-8592-a69890e03404.png)

</details>

Reply:
Done